docs/guide/cloud_concepts/cli.md

```diff
@@ -0,0 +1,371 @@
+2019-07-17T23:37:22.816988+00:00  INFO     Beginning Flow run for 'my-flow'
+2019-07-17T23:37:23.214365+00:00  INFO     Starting flow run.
+2019-07-17T23:37:23.365119+00:00  INFO     Flow 'my-flow': Handling state change from Scheduled to Running
+2019-07-17T23:37:23.915298+00:00  INFO     Task 'my_task': Starting task run...
+2019-07-17T23:37:24.113271+00:00  INFO     Task 'my_task': Handling state change from Pending to Running
+2019-07-17T23:37:24.666+00:00     INFO     Task 'my_task': Calling task.run() method...
+2019-07-17T23:37:24.813664+00:00  INFO     Task 'my_task': Handling state change from Running to Success
+2019-07-17T23:37:25.15329+00:00   INFO     Task 'my_task': finished task run for task with final state: 'Success'
+2019-07-17T23:37:25.351535+00:00  INFO     Flow run SUCCESS: all reference tasks succeeded
+2019-07-17T23:37:25.514397+00:00  INFO     Flow 'my-flow': Handling state change from Running to Success
+```
+
+To retrieve more information about each of the logs in JSON pass the `--info` flag to the command.
+```
+$ prefect get logs --name fearless-hyrax --info
+{
+    "msg": "Beginning Flow run for 'my-flow'",
+    "args": [],
+    "name": "prefect.CloudFlowRunner",
+    "msecs": 816.9882297515869,
+    "lineno": 224,
+    "module": "flow_runner",
+    "thread": 140362462062400,
+    "asctime": "2019-07-17 23:37:22,816",
+    "created": 1563406642.8169882,
+    "levelno": 20,
+    "message": "Beginning Flow run for 'my-flow'",
+    "process": 6,
+    "exc_info": null,
+    "exc_text": null,
+    "filename": "flow_runner.py",
+    "funcName": "run",
+    "pathname": "/usr/local/lib/python3.6/site-packages/prefect/engine/flow_runner.py",
+    "levelname": "INFO",
+    "stack_info": null,
+    "threadName": "MainThread",
+    "processName": "MainProcess",
+    "relativeCreated": 7002.268075942993
+}
+etc...
+```
+
+### describe
+
+`describe` is used to retrieve descriptive metadata information about Prefect Cloud objects. Currently CLI users can describe flows, flow-runs, and tasks.
+
+#### describe flows
+
+Running `prefect describe flows` requires that a flow name be provided through the `--name` option. This outputs flow descriptive metadata.
+```
+$ prefect describe flows --name my-flow
+{
+    "name": "my-flow",
+    "version": 1,
+    "project": {
+        "name": "Demo"
+    },
+    "created": "2019-07-25T14:23:21.704585+00:00",
+    "description": null,
+    "parameters": [],
+    "archived": false,
+    "storage": {
+        "type": "Docker",
+        "flows": {
+            "my-flow": "/root/.prefect/my-flow.prefect"
+        },
+        "image_tag": "9b444406-355b-4d32-832e-77ddc21d2073",
+        "image_name": "my-flow",
+        "__version__": "0.6.0",
+        "registry_url": "gcr.io/my-registry/",
+        "prefect_version": "master"
+    },
+    "environment": {
+        "type": "RemoteEnvironment",
+        "executor": "prefect.engine.executors.SynchronousExecutor",
+        "__version__": "0.6.1",
```

Comment:
🤔

Reply:
These docs will be cut in `0.6.1` so I wanted it in there for consistency